docs/advanced.rst

```diff
@@ -0,0 +1,126 @@
+Ammo providers
+==============
+
+Ammo provider is a source of test data: it makes ammo object.
+
+There is a common rule for any (built-in) provider: data supplied by ammo provider is a records that will be pushed via established connection w/ external host (defined in pandora config via `pool.gun.target` option). Thus, you cannot define to which `physical` host machine your ammo will be send using ammofile.
```

Comment:
"is a records" -> "are records"
"established connection w/ external host" -> "established connection to external host"
"will be send" -> "will be sent"
Thus, you cannot define in the ammofile to which `physical` host your ammo will be sent.